gpkit/constraints/tight.py

```diff
@@ -35,7 +35,7 @@ def process_result(self, result):
                         leftsubbed = constraint.left.sub(variables).value
                         rightsubbed = constraint.right.sub(variables).value
             if rel_diff >= self.reltol:
-                msg = ("Constraint [%.30s...] is not tight because "
+                msg = ("Constraint [%.200s...] is not tight because "
                        "the left hand side evaluated to %s but "
```

Comment:
I've found that too few characters are displayed when debugging large, linked, models. When models are linked and varkeys get tagged with a model name they become long enough that I typically only see the varkey of one variable in the constraint which makes it difficult to know which constraint isn't tight